src/autoupdater.ts

```diff
@@ -197,8 +204,8 @@ class AutoUpdater {
     return true;
   }
 
-  async merge(mergeOpts) {
-    const sleep = (timeMs) => {
+  async merge(mergeOpts: any): Promise<boolean> {
```

Comment:
You should be able to typehint this, it's whatever `this.octokit.repos.merge` takes